tutorials/multimodal/NeVA Tutorial.ipynb

```diff
@@ -0,0 +1,253 @@
+    " trainer.num_nodes=1 \\\n",
+    " trainer.devices=4 \\\n",
+    " trainer.val_check_interval=1000 \\\n",
+    " trainer.limit_val_batches=5 \\\n",
+    " trainer.log_every_n_steps=1 \\\n",
+    " trainer.max_steps=1000 \\\n",
+    " model.micro_batch_size=1 \\\n",
+    " model.global_batch_size=2 \\\n",
+    " model.tensor_model_parallel_size=4 \\\n",
+    " model.pipeline_model_parallel_size=1 \\\n",
+    " model.mcore_gpt=False \\\n",
+    " model.transformer_engine=False \\\n",
+    " model.mm_cfg.llm.from_pretrained=null \\\n",
+    " exp_manager.create_checkpoint_callback=True \\\n",
+    " model.data.data_path=/path/to/datasets/LLaVA-Pretrain-LCS-558K/blip_laion_cc_sbu_558k.json \\\n",
+    " model.data.image_folder=/path/to//datasets/LLaVA-Pretrain-LCS-558K/images \\\n",
+    " model.tokenizer.library=sentencepiece \\\n",
+    " model.tokenizer.model=/path/to/tokenizer/model \\\n",
+    " model.encoder_seq_length=4096 \\\n",
+    " model.num_layers=32 \\\n",
+    " model.hidden_size=4096 \\\n",
+    " model.ffn_hidden_size=16384 \\\n",
+    " model.num_attention_heads=32 \\\n",
+    " model.normalization=layernorm1p \\\n",
+    " model.do_layer_norm_weight_decay=False \\\n",
+    " model.apply_query_key_layer_scaling=True \\\n",
+    " model.activation=squared-relu \\\n",
+    " model.headscale=False \\\n",
+    " model.position_embedding_type=rope \\\n",
+    " model.rotary_percentage=0.5 \\\n",
+    " model.num_query_groups=null \\\n",
+    " model.data.num_workers=0 \\\n",
+    " model.mm_cfg.llm.from_pretrained=/path/to/checkpoint \\\n",
+    " model.mm_cfg.llm.model_type=nvgpt \\\n",
+    " model.data.conv_template=nvgpt \\\n",
+    " model.mm_cfg.vision_encoder.from_pretrained='openai/clip-vit-large-patch14' \\\n",
+    " model.mm_cfg.vision_encoder.from_hf=True \\\n",
+    " model.data.image_token_len=256 \\\n",
+    " model.optim.name=\"fused_adam\" \\\n",
+    " exp_manager.create_wandb_logger=False \\\n",
+    " exp_manager.wandb_logger_kwargs.project=neva_demo\n",
+    "```\n",
+    "\n",
+    "### Image-Language Pair Instruction Fine-Tuning\n",
+    "\n",
+    "Fine-tuning can also be run from within the container via a similar command leveraging the `neva_finetune.py` script.\n",
+    "\n",
+    "An example of an image-text pair instruction tuning script:\n",
+    "\n",
+    "```\n",
+    "torchrun --nproc_per_node=4 /opt/NeMo/examples/multimodal/multimodal_llm/neva/neva_finetune.py \\\n",
+    "++cluster_type=BCP \\\n",
+    " trainer.precision=bf16 \\\n",
+    " model.megatron_amp_O2=True \\\n",
+    " trainer.num_nodes=1 \\\n",
+    " trainer.devices=1 \\\n",
+    " trainer.val_check_interval=100 \\\n",
+    " trainer.limit_val_batches=50 \\\n",
+    " trainer.max_steps=4900 \\\n",
+    " model.micro_batch_size=4 \\\n",
+    " model.global_batch_size=32 \\\n",
+    " model.tensor_model_parallel_size=1 \\\n",
+    " model.pipeline_model_parallel_size=1 \\\n",
+    " model.mcore_gpt=False \\\n",
```

Comment:
same question about mcore_gpt and te

Reply:
Similar as above, `neva_finetune.yaml` has these set to `False`.

Reply:
Set both to `true` following Yu's comment